core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/ManagedGroupComponent.scala

```diff
@@ -18,13 +18,13 @@ trait ManagedGroupComponent {
   import driver.api._
 
   class ManagedGroupTable(tag: Tag) extends Table[ManagedGroupRecord](tag, "MANAGED_GROUP") {
-    def usersGroupName = column[String]("USERS_GROUP_NAME", O.Length(254), O.PrimaryKey)
-    def ownersGroupName = column[String]("OWNERS_GROUP_NAME", O.Length(254))
+    def membersGroupName = column[String]("MEMBERS_GROUP_NAME", O.Length(254), O.PrimaryKey)
+    def adminsGroupName = column[String]("ADMINS_GROUP_NAME", O.Length(254))
 
-    def * = (usersGroupName, ownersGroupName) <> (ManagedGroupRecord.tupled, ManagedGroupRecord.unapply)
+    def * = (membersGroupName, adminsGroupName) <> (ManagedGroupRecord.tupled, ManagedGroupRecord.unapply)
 
-    def usersGroup = foreignKey("FK_MANAGED_GROUP_USERS", usersGroupName, rawlsGroupQuery)(_.groupName)
-    def ownersGroup = foreignKey("FK_MANAGED_GROUP_OWNERS", ownersGroupName, rawlsGroupQuery)(_.groupName)
+    def membersGroup = foreignKey("FK_MANAGED_GROUP_MEMBERS", membersGroupName, rawlsGroupQuery)(_.groupName)
```

Comment:
Does the name of this foreign key have to match what's actually in the DB? If so, you'll need to drop and recreate both of these FKs